pkg/operator/controller/names.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"fmt"
+
+	ingressv1alpha1 "github.com/openshift/cluster-ingress-operator/pkg/apis/ingress/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	// GlobalMachineSpecifiedConfigNamespace is the location for global
+	// config.  In particular, the operator will put the configmap with the
+	// CA certificate in this namespace.
+	GlobalMachineSpecifiedConfigNamespace = "openshift-config-managed"
+
+	// caCertSecretName is the name of the secret that holds the CA certificate
+	// that the operator will use to create default certificates for
+	// clusteringresses.
+	caCertSecretName = "router-ca"
+)
+
+// RouterDeploymentName returns the namespaced name for the router deployment.
+func RouterDeploymentName(ci *ingressv1alpha1.ClusterIngress) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: "openshift-ingress",
+		Name:      "router-" + ci.Name,
+	}
+}
+
+// RouterCASecretName returns the namespaced name for the router CA secret.
+func RouterCASecretName(operatorNamespace string) types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: operatorNamespace,
+		Name:      caCertSecretName,
+	}
+}
+
+// RouterCAConfigMapName returns the namespaced name for the router CA configmap.
+func RouterCAConfigMapName() types.NamespacedName {
+	return types.NamespacedName{
+		Namespace: GlobalMachineSpecifiedConfigNamespace,
+		Name:      caCertSecretName,
```

Comment:
Did you mean to delete `caCertConfigMapName` and use `caCertSecretName` for the configmap?

Reply:
Nope, fixed. Looks like the values are the same and so it worked anyway